Sources/Extensions/UIKit/UIViewExtensions.swift

```diff
@@ -525,6 +525,42 @@ public extension UIView {
 		anchorCenterXToSuperview()
 		anchorCenterYToSuperview()
 	}
+    
+    /// SwifterSwift: Blur the background of the view
+    ///
+    /// - Parameter effectStyle: The UIBlurEffectStyle to apply to the background
+    @available(iOS 9, *) public func blurBackground(using effectStyle: UIBlurEffectStyle) {
```

Comment:
@drewpitchford Hey, thank you for your pull request 💪 🎉 Can this not be accomplished by the extensions we have on `UIView`?

Reply:
I'm not sure if it can be, but the point of this method is to wrap it up into a single method rather than having the developer have to do all the logic on his end. If you do not think it is useful, that's fine. I just found myself using this method in a lot of my projects and figured it might be useful for others.

Reply:
I see. We already have this extension on `UIImageView`. I was thinking this was on `UIView`. It would be good to provide this extension on all `UIView` classes. @omaralbeik Is there any reason why this extension is on `UIImageView` opposed to `UIView`?

Reply:
@SD10 @drewpitchford Nope, I don't remember there was a reason for this, @drewpitchford it would be great to submit another PR with only `blurBackground` moved to UIView to separated it from other changes